simplex.py

```diff
@@ -170,6 +170,8 @@ def run(self, r_data=None):
                 ff.score = compare.compare_data(r_data, data)
                 opt.pretty_ff_results(ff)
         self.new_ffs = sorted(self.new_ffs + [ff_copy], key=lambda x: x.score)
+        # Allow 3 cycles w/o change for each parameter present
+        self.max_cycles_wo_change = 3*(len(self.new_ffs)-1)
```

Comment:
Right, it's non-private, which means that if a user sets it in `__init__`, he expects that it will actually be used.

If you overwrite it, it's acting like a variable for internal use, thus it should be lead by one underscore.

It is not a private variable, so it also should not have doubling leading underscores.